controllers/nodes.go

```diff
@@ -259,7 +260,8 @@ func (n *Nodes) Reconcile(ctx context.Context, clt client.Client, scheme *runtim
 	log := ctrllog.FromContext(ctx)
 
 	if n.Enable {
-		mondooImage, err := resolveMondooImage(log, n.Mondoo.Spec.Nodes.Image.Name, n.Mondoo.Spec.Nodes.Image.Tag)
+		resolveImage := n.MondooOperatorConfig.Spec.SkipContainerResolution
```

Comment:
The name of the variable sounds reversed to its actual purpose
```suggestion
		skipResolveImage := n.MondooOperatorConfig.Spec.SkipContainerResolution
```